docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -0,0 +1,216 @@
+the categories that represent the highest and lowest values. {kib} also supports vertical bar charts.
+
+| image:images/bar.png[Bar chart]
+
+a| *Line*
+
+Displays data points that are connected by a line. Use line charts to visualize a sequence of values, discover
+trends over time, and forecast future values.
+
+| image:images/line.png[Line chart]
+
+a| *Metric*
+
+Displays a single numeric value for an aggregation. Use the metric visualization when you have a numeric value that is powerful enough to tell
+a story about your data.
+
+| image:images/metric.png[Metric]
+
+a| *Pie*
+
+Displays slices that represent a data category, where the slice size is proportional to the quantity it represents.
+Use pie charts to show comparisons between multiple categories, illustrate the dominance of one category over others,
+and show percentage or proportional data.
+
+| image:images/pie.png[Pie chart]
+
+a| *Tag cloud*
+
+Graphical representations of how frequently a word appears in the source text. Use tag clouds to easily produce a summary of large documents and
+create visual art for a specific topic.
+
+| image:images/tag_cloud.png[Tag cloud]
+
+|===
+
+[float]
+[[create-aggregation-based-panel]]
+=== Create an aggregation-based panel
+
+Choose the type of panel you want to create, then use the editor to configure the options. Each panel type supports different options. 
+
+. From the dashboard, click *Create new*, then click *Aggregation based* on the *New visualization* window.
+
+.. Click the type of panel you want to create.
+
+.. Click the data source you want to visualize.
+
+. From the editor, add the <<aggregation-reference,aggregations>> you want to visualize, then click *Update*. 
++
+NOTE: For the *Date Histogram* to use an *auto interval*, the date field must match the primary time field of the index pattern.
+
+. To change the order, drag the aggregations along the editor. 
++ 
+[role="screenshot"]
+image:images/bar-chart-tutorial-3.png[Option to change the order of aggregations]
+
+. To customize the series colors, click the series in the legend, then select the color you want to use.
++ 
+[role="screenshot"]
+image:images/aggregation-based-color-picker.png[Color picker]
+
+[float]
+[[try-it-aggregation-based-panel]]
+=== Try it: Create an aggregation-based bar chart
+
+You collected data from your web server, and you want to visualize and analyze the data on a dashboard. To create a dashboard panel of the data, create
+a bar chart that displays the top five log traffic sources for every three hours.
+
+[float]
+==== Add the data and create the dashboard
+
+Add the sample web logs data that you'll use to create the bar chart, then create the dashboard.
+
+. From the {kib} *Home* page, click *Try our sample data*.
+
+. From *Sample web logs*, click *Add data*.
+
+. Open the main menu, then click *Dashboard*.
+
+. Click *Create dashboard*.
```

Comment:
When I click Dashboard, I'm navigated directly to the dashboard.  I don't see a Create dashboard button.

Reply:
That's what happens if you previously had an open dashboard. We should probably indicate that there are 2 pages: the list of dashboards and a dashboard view